components/movementsensor/gpsnmea/nmeaParser.go

```diff
@@ -238,8 +238,9 @@ func (g *GPSData) updateGNS(gns nmea.GNS) error {
 	return nil
 }
 
-//nolint:all
 // updateHDT updaates g.CompassHeading with the ground speed information from the provided
+//
+//nolint:all
```

Comment:
Eh... any chance this can be made more specific? Which lint rules are we trying to skip here?

I think of `nolint:all` as being a quick-and-dirty thing while you're debugging, and the ones that end up in the final code should be only the rules we're skipping on purpose.